src/mobile/checkbox/index.ts

```diff
@@ -276,7 +276,7 @@ export class MobileCheckbox extends KucBase {
     return html`
       <style>
         kuc-mobile-checkbox,
-        kuc-mobile-checkbox * {
+        kuc-mobile-checkbox {
```

Comment:
Why do you need to double string "kuc-mobile-checkbox, kuc-mobile-checkbox" ???
I think we only need kuc-mobile-checkbox {....} is enough, if it is right, please update code and spec
Thank you!